transcom/mymove
# B 18990 mto shipment locator integration

## [B-18990](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A909786)

## Summary

This PR adds a shipment locator to each shipment under the MTO tab for any officer.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Use existing or create any move
2. Approve move as SC
3. Approve items as TOO
3. Login as SC, TOO, TIO, SC/Closeout SC, or QAE
4. Click "Move task order" tab and verify shipment ID to be present

### Frontend


## Screenshots
<img width="1050" alt="Screenshot 2024-04-16 at 12 35 25 PM" src="https://github.com/transcom/mymove/assets/146969726/f3acb1db-a3ce-45ee-91f9-2f76201b4322">
